sort/gnome.go

```diff
@@ -1,5 +1,9 @@
 package sort
 
+
+// The gnome sort is a sorting algorithm which is similar
```

Comment:
Thanks. Another minor thing.
The comment always start with the function, variable, etc name.
So it should be "GnomeSort is a sorting algorithm...."

From [GoDoc Documentation](https://blog.golang.org/godoc-documenting-go-code):
> Notice this comment is a complete sentence that begins with the name of the element it describes. This important convention allows us to generate documentation in a variety of formats, from plain text to HTML to UNIX man pages, and makes it read better when tools truncate it for brevity, such as when they extract the first line or sentence.

Reply:
Is that ok ?

Reply:
perfect.